modules/machineset-vsphere-required-permissions.adoc

```diff
@@ -76,6 +76,11 @@ If you cannot use an account with global administrative privileges, you must cre
 
 3+a|
 ^1^ The `StorageProfile.Update` and `StorageProfile.View` permissions are required only for storage backends that use the Container Storage Interface (CSI).
+
+[IMPORTANT]
+=====
+Some CSI drivers and features are in Technology Preview in {product-title} {product-version}. For more information, see _CSI drivers supported by OpenShift Container Platform_.
```

Comment:
Could we add a link here as well? In the doc, it appears in front of "creating-machineset-vsphere.adoc"
Others looks good to me.

Reply:
@duanwei33 thank you for your review. At the moment, cross-references are not supported in modules. Instead, I have added the title here and then included the cross reference in the assembly that contains this module.

Reply:
Thanks, looks good to me. @pneedle-rh